keps/prod-readiness/sig-network/3015.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group 
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 3015
+alpha:
+  approver: "@johnbelamaric"
```

Comment:
Given that Jon didn't yet take a look and I have a context, please reassign to me.